src/screens/Notification/Notification.tsx

```
@@ -152,23 +156,24 @@ const Notification: React.FC = () => {
-            disabled={notifications.length < pageSize}
-          >
-            Next
-          </button>
-        </div>
+        {(page > 0 || notifications.length > 1) && (
+          <div className={styles.paginationFooter}>
+            <button
+              className={styles.paginationButton}
+              onClick={handlePrev}
+              disabled={page === 0}
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)